src/index.js

```diff
@@ -0,0 +1,143 @@
+"use strict";
+
+const increaseTemperature = () => {
+    state.temperature += 1;
+    updateColorLandscape();
+}
+
+const decreaseTemperature = () => {
+    state.temperature -= 1;
+    updateColorLandscape();
+}
+
+
+const state = {
+    temperature: 68,
+    city: 'Montreal'
+};
+
+
+function updateColorLandscape() {
+    const landscape = document.getElementById("landscape");
+    const tempValue = document.getElementById("tempValue");
+    tempValue.textContent = state.temperature;
+
+    if (state.temperature >= 80) {
+        tempValue.className = "red";
+        landscape.textContent = "🌵__🐍_🦂_🌵🌵__🐍_🏜_🦂";
+    } else if (state.temperature >= 70 && state.temperature <= 79) {
+        tempValue.className = "orange";
+        landscape.textContent = "🌸🌿🌼__🌷🌻🌿_☘️🌱_🌻🌷";
+    } else if (state.temperature >= 60 && state.temperature <= 69) {
+        tempValue.className = "yellow";
+        landscape.textContent = "🌾🌾_🍃_🪨__🛤_🌾🌾🌾_🍃";
+    } else if (state.temperature >= 50 && state.temperature <= 59) {
+        tempValue.className = "green";
+        landscape.textContent = "🌲🌲⛄️🌲⛄️🍂🌲🍁🌲🌲⛄️🍂🌲";
+    } else if (state.temperature <= 49) {
+        tempValue.className = "teal";
+        landscape.textContent = "🌲🌲⛄️🌲⛄️🍂🌲🍁🌲🌲⛄️🍂🌲";
+    }
+    headerCityName.textContent = cityNameInput.value;
+}
+
+
+increaseTempControl.addEventListener("click", increaseTemperature);
+decreaseTempControl.addEventListener("click", decreaseTemperature);
+cityNameInput.addEventListener("input", updateColorLandscape);
+currentTempButton.addEventListener("click", function() {
+    findLatitudeLongitude()
+});
+
+
+cityNameReset.addEventListener("click", function() {
+    cityNameInput.value = "Montreal";
+    state.temperature = 68;
+    updateColorLandscape();
+});
+
+
+
+//dropdown
+const skySelect = document.getElementById("skySelect");
+skySelect.addEventListener("change", function() {
+    const selectedSky = skySelect.value;
+
+    if (selectedSky === "sunny") {
+        gardenContent.textContent = "☁️ ☁️ ☁️ ☀️ ☁️ ☁️";
+        gardenContent.className = "sunny"
+        console.log("☁️ ☁️ ☁️ ☀️ ☁️ ☁️");
+    } else if (selectedSky === "cloudy") {
+        gardenContent.textContent = "☁️☁️ ☁️ ☁️☁️ ☁️ 🌤 ☁️ ☁️☁️";
+        gardenContent.className = "cloudy"
+        console.log("☁️☁️ ☁️ ☁️☁️ ☁️ 🌤 ☁️ ☁️☁️");
+    } else if (selectedSky === "rainy") {
+        gardenContent.textContent = "🌧🌈⛈🌧🌧💧⛈🌧🌦🌧💧🌧🌧";
+        gardenContent.className = "rainy"
+        console.log("🌧🌈⛈🌧🌧💧⛈🌧🌦🌧💧🌧🌧");
+    } else if (selectedSky === "snowy") {
+        gardenContent.textContent = "🌨❄️🌨🌨❄️❄️🌨❄️🌨❄️❄️🌨🌨";
+        gardenContent.className = "snowy"
+        console.log("🌨❄️🌨🌨❄️❄️🌨❄️🌨❄️❄️🌨🌨");
+    }
```

Comment:
Another candidate for an object of key-value pairs with the keys as the sky conditions and the value is the emojis